def/fb-harmony.js

```diff
@@ -150,7 +150,7 @@ def("FunctionTypeParam")
   .field("name", def("Identifier"))
   .field("typeAnnotation", def("Type"))
   .field("optional", isBoolean);
-  
```

Comment:
Apparently my editor stripped trailing white spaces. I can revert that if you want.